wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/RevertResult.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.git.shared;
+
+import java.util.List;
+import java.util.Map;
+import org.eclipse.che.dto.shared.DTO;
+
+/** @author Dmitrii Bocharov (bdshadow) */
+@DTO
+public interface RevertResult {
+
+  public enum RevertStatus {
+    FAILED("Failed to revert"),
+    DIRTY_INDEX("The revert failed because of a dirty index"),
+    DIRTY_WORKTREE("The revert failed because of a dirty working tree"),
+    COULD_NOT_DELETE("The revert failed because a file could not be deleted");
+
+    private final String value;
+
+    private RevertStatus(String value) {
+      this.value = value;
+    }
+
+    public String getValue() {
+      return value;
+    }
+  }
+
+  /** @return the conflicted files with revert status (reason of failure) */
+  Map<String, RevertStatus> getConflicts();
+
+  void setConflicts(Map<String, RevertStatus> conflicts);
+
+  RevertResult withConflicts(Map<String, RevertStatus> conflicts);
+
+  /** @return the successfully reverted list of files */
```

Comment:
I prefer not to have single row with `@return` annotation in Javadocs. I think it would be better to write `Returns ...`

Reply:
anyway finally the result is almost the same. `@return` only makes the "Return" word bold. From the source code point of view, there is no difference too.
Plus this format is used in most of the classes in `org.eclipse.che.api.git.shared` package.